pulp_ansible/app/migrations/0030_collectionversion_requires_ansible.py

```diff
@@ -25,20 +25,27 @@ def set_requires_ansible_and_manifest_and_files_json(apps, schema_editor):
                     runtime_yaml = yaml.safe_load(runtime_metadata)
                 except YAMLError:
                     log.warning(
-                        "CollectionVersion: {namespace}.{name}-{version} - meta/runtime.yml is invalid yaml".format(
+                        "CollectionVersion: '{namespace}.{name}-{version}' - 'meta/runtime.yml' is invalid yaml.".format(
                             namespace=collection_version.namespace, name=collection_version.name, version=collection_version.version
                         )
                     )
                 else:
-                    collection_version.requires_ansible = runtime_yaml.get("requires_ansible")
+                    try:
+                        collection_version.requires_ansible = runtime_yaml.get("requires_ansible")
```

Comment:
From the error message linked in the issue it seems like the `yaml.safe_load` returns a `None` object.  `get` typically returns `None` if the key is not present, so could we just change this to check if `runtime_yaml` is `None` and then call `get`?

Reply:
What's the benefit? I'd consider it to be an exception if the collection is too old to know about these fields...